### PR TITLE
[GPU] Fix convert color I420

### PR DESCRIPTION
**Pull request by andrei-cv**
### Details:
 - *fix convert color i420*

### Tickets:
 - *98546*

### CODE REVIEW

File: src/tests/functional/plugin/gpu/shared_tests_instances/skip_tests_config.cpp [hunk 1]
```diff
@@ -125,6 +125,6 @@ std::vector<std::string> disabledTestPatterns() {
             R"(.*smoke_GroupConvolution1D_ExplicitPadding_Disabled.*)",
             R"(.*smoke_GroupConvolutionLayerGPUTest_dynamic1DSymPad_Disabled.*)",
             R"(.*smoke_ConvolutionLayerGPUTest_dynamic1DSymPad.*)",
-            R"(.*TestsConvertColorI420.*)",
+            //R"(.*TestsConvertColorI420.*)",
```
Comment (vladimir-paramuzov) [inline src/tests/functional/plugin/gpu/shared_tests_instances/skip_tests_config.cpp:128]: delete it

**Review by ilyachur — APPROVED** [review]

**Review by vladimir-paramuzov — APPROVED** [review]

**Pull request merged**